agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml

```
@@ -134,6 +137,9 @@ spec:
 #        - mountPath: /host/etc/VERSION
 #          name: etc-version
 #          readOnly: true
+        - mountPath: /host/etc
+          name: etc-vol
+          readOnly: true
```

Personally, I would only mount `/etc/passwd` and `/etc/groups` to keep the scope as small as possible and being more consistent with the rest of the manifest (e.g. https://github.com/draios/sysdig-cloud-scripts/blob/e9c50731ef58f2c9627f776939f9129c11eb4173/agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml#L168-L170).
```suggestion
        - mountPath: /host/etc/passwd
          name: passwd-vol
          readOnly: true
       - mountPath: /host/etc/groups
          name: groups-vol
          readOnly: true
```
The volumes should be defined too.

---

That's not working, actually.
Bind mount of single files don't stay in sync after the inode change of the file, which is what e.g. `useradd` does.

---

You're right good point. As now it is superfluous, I would just remove the following as we discussed offline.

https://github.com/draios/sysdig-cloud-scripts/blob/e9c50731ef58f2c9627f776939f9129c11eb4173/agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml#L24-L27

https://github.com/draios/sysdig-cloud-scripts/blob/e9c50731ef58f2c9627f776939f9129c11eb4173/agent_deploy/kubernetes/sysdig-agent-daemonset-v2.yaml#L168-L170

---

removed `osrel`